ships/augur.json

```diff
@@ -87,6 +87,7 @@
               }
             ]
           },
+          "id": 0,
```

Comment:
The only usage I can think of for an ID, would be an outside reference. Ie: A system referring to a hostile, or a system referring to an armada. I don't really see what an ID would be doing in a tiered component, tbh.

Reply:
I'll remove that field from the files and the schema

Reply:
The ID was used since I normally use 'name' as the unique field for every building, research, officer, ship, ... since they are always unique. But components can have multiple weapons with the same name. That's why ID was added. But if we decrease the components and use quantity, the names are again unique.